northstar/src/runtime/fork/init/builder.rs

```diff
@@ -88,11 +92,13 @@ async fn prepare_mounts(
     config: &Config,
     root: &Path,
     manifest: &Manifest,
+    containers: &(impl Iterator<Item = Container> + Clone),
 ) -> Result<Vec<Mount>, Error> {
     let mut mounts = vec![];
     let manifest_mounts = &manifest.mounts;
 
     for (target, mount) in manifest_mounts {
+        let mut containers = containers.clone();
```

Comment:
Why is this clone needed?

Reply:
Every iteration needs a fresh copy of the iterator as `State::match_container` will modify it. I moved the cloning into match_container to make it more clear.